torch/onnx/_internal/onnxruntime.py

```diff
@@ -797,6 +798,18 @@ def maybe_map_to_meta_val(value):
                 opset_version=self._resolved_onnx_exporter_options.onnx_registry.opset_version,
             )
 
+            # Inline model before executing in ORT.
+            # Temporary solution to improve performance until runtimes adapt to function format.
```

Comment:
Adding a TODO comment with the issue URL would help the future us to track and remove this workaround when the time is right